component/attr_cache/attr_cache.go

```diff
@@ -662,6 +662,27 @@ func (ac *AttrCache) CreateFile(options internal.CreateFileOptions) (*handlemap.
 	return h, err
 }
 
+// OpenFile: Update cache with Open results from cloud
+func (ac *AttrCache) OpenFile(options internal.OpenFileOptions) (*handlemap.Handle, error) {
```

Comment:
Since you are adding a new function to attr_cache, could you add some unit tests for it?

Reply:
Good point!
This now has a brand new unit test.